Cognite.Simulator.Utils/SimulationRunnerBase.cs

```diff
@@ -439,25 +442,39 @@ async void PublishSimulationRunStatus(string runStatus, CancellationToken token)
                     configObj,
                     validationEnd);
             }
-            // Run the simulation
-            await RunSimulation(
-                simEv,
-                startTime,
-                modelState,
-                configState,
-                configObj,
-                samplingRange,
-                token).ConfigureAwait(false);
+            try
+            {
+                 // Run the simulation
+                await RunSimulation(
+                    simEv,
+                    startTime,
+                    modelState,
+                    configState,
+                    configObj,
+                    samplingRange,
+                    token).ConfigureAwait(false);
 
-                simEv.Run = await UpdateSimulationRunStatus(
-                    simEv.Run.Id,
-                    SimulationRunStatus.success,
-                    "Calculation ran to completion",
-                    token,
-                    simEv.RunConfiguration
-                ).ConfigureAwait(false);
+                    simEv.Run = await UpdateSimulationRunStatus(
+                        simEv.Run.Id,
+                        SimulationRunStatus.success,
+                        "Calculation ran to completion",
+                        token,
+                        simEv.RunConfiguration
+                    ).ConfigureAwait(false);
 
-            await EndSimulationRun(simEv, token).ConfigureAwait(false);
+                await EndSimulationRun(simEv, token).ConfigureAwait(false);
+            }
+            catch (Exception ex)
+            {
+                
+                Console.WriteLine("Error in RunSimulation");
+                Console.WriteLine(ex);
+            }
```

Comment:
```suggestion
            catch (Exception ex)
            {
                
                throw;
            }
```
we need to rethrow the exception here